Added detailed documentation on how to start with an ent license (#578)
hazelcast/management-center-docs: docs/modules/getting-started/pages/get-started.adoc

+
`MC Client connected to cluster dev.`

=== Start the compose stack with an enterprise license

> If you own an enterprise license you can configure the license in the compose file as following (see xref:deploy-manage:system-properties.adoc#starting-with-a-license[`hazelcast.mc.license` property] for more details):

Reviewer: ```suggestion
If you have an Enterprise Edition license, you can configure the Docker Compose file with the license as follows (see xref:deploy-manage:system-properties.adoc#starting-with-a-license[`hazelcast.mc.license` property] for more details):
```

Reviewer: I'm assuming that it's the Docker compose file that's configured with the license?

ifdef::snapshot[]
[source,yaml,subs="attributes+"]
----
services:
  hazelcast:
    image: hazelcast/hazelcast-enterprise:latest
    ports:
    - "5701:5701"
    environment:
    - HZ_LICENSEKEY={license key}
  management-center:
    image: hazelcast/management-center:{full-version}
    ports:
    - "8080:8080"
    environment:
    - MC_LICENSE={license key}
----
endif::[]
ifndef::snapshot[]
[source,bash,subs="attributes+"]
----
services:
  hazelcast:
    image: hazelcast/hazelcast-enterprise:latest
    ports:
    - "5701:5701"
    environment:
    - HZ_LICENSEKEY={license key}
  management-center:
    image: hazelcast/management-center:{full-version}
    ports:
    - "8080:8080"
    environment:
    - MC_LICENSE={license key}
    - MC_DEFAULT_CLUSTER=dev
    - MC_DEFAULT_CLUSTER_MEMBERS=hazelcast
----

> endif::[]

Reviewer 2: Also - can we include this is a shared snippet between both pages?

> endif::[]

Author: good catch - I have blindly copied the section on line :43 - I'll ask in the channel

> endif::[]

Reviewer 2: > This change made both sections equal. Not sure what is the right approach here now but probably we can get rid of the conditional.

There is another difference:
```
    - MC_DEFAULT_CLUSTER=dev
    - MC_DEFAULT_CLUSTER_MEMBERS=hazelcast
```

> endif::[]

Reviewer 3: I meant the sections from line 43 of the existing code
ifdef::snapshot[]
ifndef::snapshot[]

<img width="626" height="726" alt="Image" src="https://github.com/user-attachments/assets/382e6a8f-8471-4821-ac78-485ca4ba4614" />

The new code obviously has more differences with the existing one:
1.  `image: hazelcast/hazelcast-enterprise:latest`
 vs existing 
`image: hazelcast/hazelcast:latest`
2. Platform license key
3. MC license key
4. And the one you've mentioned: 
```
  - MC_DEFAULT_CLUSTER=dev
    - MC_DEFAULT_CLUSTER_MEMBERS=hazelcast
```
I believe the latter one should be in both `ifdef` and `ifndef`. And these "both" should be essentially the same.

But in terms of having a common snippet between line 43 block and the new code - they are quite different.
